contribs/gmf/src/controllers/abstractdesktop.js

```diff
@@ -3,7 +3,7 @@ goog.provide('gmf.AbstractDesktopController');
 goog.require('gmf');
 goog.require('gmf.AbstractController');
 /** @suppress {extraRequire} */
-goog.require('gmf.mobileBackgroundLayerSelectorDirective');
+goog.require('gmf.mobilebackgroundlayerselectorDirective');
```

Comment:
Is mobilebackgroundlayerselectorDirective really useful in the desktop file ? Or can I retire it ?